ush/ufsda/yamltools.py

```diff
@@ -69,13 +69,18 @@ def parse_config(input_config_dict, template=None, clean=True):
 def atmanl_case(config):
     # compute atm analysis case/res variables based on environment and/or config
     case = int(config.get('CASE', os.environ.get('CASE', 'C768'))[1:])
+    case_anl = int(config.get('CASE_ANL', os.environ.get('CASE_ANL', 'C384'))[1:])
     case_enkf = int(config.get('CASE_ENKF', os.environ.get('CASE_ENKF', 'C384'))[1:])
     levs = int(config.get('LEVS', os.environ.get('LEVS', '128')))
     if 'DOHYBVAR' in config:
         dohybvar = config['DOHYBVAR']
         del config['DOHYBVAR']
     else:
         dohybvar = isTrue(os.environ.get('DOHYBVAR', 'NO'))
+    # if dohybar is true, we currently need to ensure case_enkf = case_anl
+    if dohybvar and not case_enkf == case_anl:
+        raise KeyError(f"dohybvar is '{dohybvar}' but case_enkf= '{case_enkf}' does not equal case_anl= '{case_anl}'")
```

Comment:
Nit picking, but I don't think this is a 'key error' (that is reserved for missing/wrong keys in a dictionary). I think this would be a ValueError perhaps? (case_enkf == case_anl, so the values are inconsistent?)

Reply:
Thanks.  I'm not familiar with python lingo.   What you say makes sense.  This is a value error.  I'll change `KeyError` to `ValueError`.

I don't like having two variables, `case_anl` and `case_enkf` with the same value when `dohybvar` is true.  One variable would suffice.   However, `case_enkf` has uses in g-w beyond variational DA.  `case_enkf` is an appropriate name for these other uses.